Swift/Tests/CBLTestCase.swift

```diff
@@ -199,6 +205,18 @@ class CBLTestCase: XCTestCase {
         }
     }
     
+    
+    func ignoreExpcetion(block: @escaping () throws -> Void) {
```

Comment:
```suggestion
    func ignoreException(block: @escaping () throws -> Void) {
```

Reply:
Thanks. Will fix this.

Reply:
I think the implementation of the method can be simplified too. I will do that.